step/src/main/java/com/google/sps/Notification.java

```diff
@@ -15,5 +15,56 @@
 package com.google.sps;
 
 public abstract class Notification {
-  // TODO: implement this class    
-}
\ No newline at end of file
+  private long id;  
+  protected String otherUser;
```

Comment:
Would it be better to store the id of the other user? I think an id would allow us more options like creating a hyperlink in the notification.

Reply:
I agree with that. It would be easier to send the match notification to both the user and the other user when two people get matched.

Reply:
Sure, can do.